ext/curl/curl.stub.php

```diff
@@ -407,6 +407,13 @@
  * @cvalue CURLOPT_STDERR
  */
 const CURLOPT_STDERR = UNKNOWN;
+#if LIBCURL_VERSION_NUM >= 0x080801 /* Available since 8.9.0 */
```

Comment:
Isn't this supposed to be `0x080900`? (same in interface.c)

Reply:
yes